compiler-cli/src/fs.rs

```diff
@@ -27,11 +27,48 @@ use crate::{dependencies::UseManifest, lsp::LspLocker};
 #[cfg(test)]
 mod tests;
 
+/// Return the current directory as a UTF-8 Path
+/// # Panics
+/// This function will panic if it fails to determine the current directory,
+/// or if your current directory is not a UTF8 Path.
```

Comment:
This function doesn't panic?

Reply:
It did when I wrote the comment and I forgot to update it xD

Reply:
Changing this to return a `FileIoError`. What would the `FileIoAction` be for failing to locate the current directory? Open?